webapi/src/main/scala/org/knora/webapi/routing/admin/ListsRouteADM.scala

```diff
@@ -66,7 +68,24 @@ object ListsRouteADM extends Authenticator {
             } ~
             post {
                 /* create a list */
-                ???
+                entity(as[CreateListApiRequestADM]) { apiRequest =>
```

Comment:
If the request is syntactically invalid (the JSON can't be parsed, or it's parsed but contains the wrong information), what sort of error does the client get?

Reply:
No idea. I will check and write a test.

Reply:
Added payload testing inside `CreateListApiRequestADM` and `ChangeListInfoApiRequestADM `. There are now also additional tests in `ListsMessagesADMSpec`